jujugui/static/gui/src/app/components/profile/navigation/_navigation.scss

```diff
@@ -0,0 +1,12 @@
+.profile-navigation {
+
+  li {
```

Comment:
You should target class names in components. So the underlying markup can change or be different in other places of the app without breaking the styling.

For example: `&__list-item`.